docs/users/basics/transactions.md

```diff
@@ -36,7 +36,53 @@ Explain that transactions can interoperate with other blockchains.
 
 ### Ethereum Transactions
 
-<!-- TODO: transactions that interact with the EVM -->
+Evmos supports the following Ethereum transactions.
+
+#### Legacy Transactions
+
+Legacy transactions are the transaction format used prior to [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718), the Ethereum Improvement Proposal which transitioned from [RLP-encoded](https://ethereum.org/en/developers/docs/data-structures-and-encoding/rlp/) transactions to a new generalized envelope for typed transactions.
+
+Ethereum legacy transactions take the following form:
+
+```bash
+RLP([nonce, gasPrice, gasLimit, to, value, data, v, r, s])
+```
+
+while the new standard for transactions takes this form:
+
+```bash
+TransactionType || TransactionPayload
+```
+
+where `TransactionType` is a number between `0x0` and `0x7f`, for a total of 128 possible transaction types, and `TransactionPayload` is an arbitrary byte array, defined by the transaction type.
+
+EIP-2718 is backwards-compatible, meaning legacy transactions are still valid as transactions on the Ethereum network. RLP-encoded transactions always begin with a byte larger than or equal to `0xc0`, so typed transactions do not collide with legacy transactions, and differentiating between them is simple.
+
+Evmos supports not only legacy transactions, but several typed transactions ([see below](#access-list-transactions-eip-2930httpseipsethereumorgeipseip-2930))!
+
+#### Access List Transactions ([EIP-2930](https://eips.ethereum.org/EIPS/eip-2930))
+
+Evmos supports access list transactions, which were introduced in EIP-2930 and take the following form:
+
+```bash
+0x01 || RLP([chainId, nonce, gasPrice, gasLimit, to, value, data, accessList, signatureYParity, signatureR, signatureS])
+```
+
+This transaction type contains an `accessList`, which is a list of addresses and storage keys that the transaction plans to access. Gas costs for transactions are still charged, but at a discount relative to the cost of accessing outside the list.
+
+#### Dynamic Fee Transactions ([EIP-1559](https://eips.ethereum.org/EIPS/eip-1559))
```

Comment:
The ordering of sections should be from newest tx type to oldest

- Dynamic Fee
- Access List
- Legacy Tx